edge/client/portainer_edge_async_client.go

```diff
@@ -438,7 +438,34 @@ func (client *PortainerAsyncClient) DeleteEdgeStackStatus(edgeStackID int) error
 
 // GetEdgeStackConfig retrieves the configuration associated to an Edge stack
 func (client *PortainerAsyncClient) GetEdgeStackConfig(edgeStackID int) (*agent.EdgeStackConfig, error) {
-	return nil, nil // unused in async mode
+	requestURL := fmt.Sprintf("%s/api/endpoints/%d/edge/stacks/%d", client.serverAddress, client.getEndpointIDFn(), edgeStackID)
```

Comment:
Async mode **MUST NOT** make any extra requests to Portainer, all the information exchange needs to happen via the async polling loop, which uses `/endpoints/edge/async`. This is a strict requirement.